docs/connected-accounts.rst

```diff
@@ -13,8 +13,7 @@ Connecting your account allows for:
 If you signed up or logged in to Read the Docs with your GitHub, Bitbucket, or GitLab
 credentials, you're all done. Your account is connected.
 
-To connect your unconnected account, go to your *Settings* dashboard
-and select `Connected Services <https://readthedocs.org/accounts/social/connections/>`_.
+To connect a social account, go to your :guilabel:`Username dropdown` > :guilabel:`Settings` > :guilabel:`Connected Services`.
```

Comment:
I think we should still mention the link, something like "go to this link or ...". It is easier to just click than search for the UI elements.

Reply:
We need a way to link to both sites, if we're providing a link..